_includes/core/head.html

```diff
@@ -5,17 +5,17 @@
   <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
   <title>{{ page.title }}</title>
 
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
 <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
 <link rel="stylesheet" href="../styles/coreclr/coreclr.css">
-<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/animate.css/3.2.6/animate.min.css">
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/animate.css/3.2.6/animate.min.css">
 
 <!-- Custom styles for this template -->
 <link href="http://getbootstrap.com/examples/starter-template/starter-template.css" rel="stylesheet">
```

Comment:
:exclamation: I don't think hotlinking to this file is encouraged by bootstrap since it's not on the CDN, we should just copy it into the repo.